File: extractor-core/src/main/java/ru/mydesignstudio/database/metadata/extractor/extractors/model/ForeignKeyModel.java
package ru.mydesignstudio.database.metadata.extractor.extractors.model;

import lombok.Data;

@Data
public class ForeignKeyModel {
  private String foreignKey;
  private String constraintType;
  private String schemaName;
  private String constraintName;
  private String tableName;
  private String columns;
  private String rOwner;
  private String rConstraintName;
}

File: extractor-core/src/main/kotlin/ru/mydesignstudio/database/metadata/extractor/extract/result/ForeignKeyModel.kt
package ru.mydesignstudio.database.metadata.extractor.extract.result

class ForeignKeyModel(
        val foreignKey: String = "",
        val constraintType: String = "",
        val schemaName: String = "",
        val constraintName: String = "",
        val tableName: String = "",
        val columns: String = "",
        val rOwner: String = "",
        val rConstraintName: String = ""
) {
}